### ISSUE TITLE
Spleef couldn't find a permissions plugin

### ISSUE THREAD

**Issue by Biogemuese** [Kaos1337/SimpleSpleef-Evolution#6]
I'm using permissionsbukkit and spleef doesnt work. "...spleef doesnt find a permissions plugin..."

**Comment by aaomidi** [comment]
Can you post the full log?

**Comment by Biogemuese** [comment]
http://pastebin.com/XQur2dum

**Comment by aaomidi** [comment]
I think that permissions bukkit is either not supported or you haven't enabled it, Try using either Group Manager (Recommended) or PEX.

**Comment by AldinVotch** [comment]
I use PEX and i have the same issue.

**Comment by aaomidi** [comment]
Yeah I noticed that the problem still remains with PEX, just use group manager for now :P
Also I should say PEX vs GroupManager, GM wins.

**Comment by zorlax** [comment]
The Vault (http://dev.bukkit.org/server-mods/vault/) plugins is required to run SimpleSpleef
